seed/tests/test_greenbutton_import.py

```diff
@@ -100,14 +100,14 @@ def test_green_button_import_base_case(self):
         self.assertEqual(meter_reading_10.start_time, make_aware(datetime(2011, 3, 5, 21, 0, 0), timezone=self.tz_obj))
         self.assertEqual(meter_reading_10.end_time, make_aware(datetime(2011, 3, 5, 21, 15, 0), timezone=self.tz_obj))
         self.assertAlmostEqual(meter_reading_10.reading, 1790 * 3.41 / 1000)
-        self.assertEqual(meter_reading_10.source_unit, 'kWh (thousand Watt-hours)')
-        self.assertEqual(meter_reading_10.conversion_factor, 3.41)
+        self.assertEqual(meter_reading_10.source_unit, 'Wh (Watt-hours)')
```

Comment:
just curious, do you know why these tests for greenbutton changed? I would assume they wouldn't be affected by BSync stuff

Reply:
If you load up the file it's importing you will notice the meters are reporting a uom code of '72' which is just watt hours (see seed/lib/mcm/reader.py).  However, 'Wh' weren't supported previously, so they were being converted to 'kWh' and all the tests had to mirror that.  Now that the parser can support Wh directly, the tests had to be updated.